.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+---
+exclude:
+  '[bootstrap.py|tests/templates/*]'
+repos:
+    - repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v3.4.0
+      hooks:
+          - id: trailing-whitespace
+          - id: end-of-file-fixer
+          - id: fix-encoding-pragma
+            args: [--remove]
+          - id: check-yaml
+          - id: debug-statements
+            language_version: python3
+    - repo: https://gitlab.com/pycqa/flake8
+      rev: 3.8.4
+      hooks:
+          - id: flake8
+            language_version: python3
+            additional_dependencies: [flake8-typing-imports==1.9.0]
+    - repo: https://github.com/pre-commit/mirrors-autopep8
+      rev: v1.5.4
+      hooks:
+          - id: autopep8
+    - repo: https://github.com/timothycrosley/isort
+      rev: 5.6.4
+      hooks:
+      - id: isort
+        args: [--filter-files]
+        files: \.py$
+    - repo: https://github.com/asottile/add-trailing-comma
+      rev: v2.0.2
+      hooks:
+          - id: add-trailing-comma
```

Comment:
I do not like it at all and I do not buy the argumentation why it might be useful.